internal/pkg/cli/job_init.go

```diff
@@ -84,6 +87,50 @@ func newInitJobOpts(vars initJobVars) (*initJobOpts, error) {
 
 // Validate returns an error if the flag values passed by the user are invalid.
 func (o *initJobOpts) Validate() error {
+	if o.Name != "" {
+		if err := validateJobName(o.Name); err != nil {
+			return err
+		}
+	}
+	if o.DockerfilePath != "" {
+		if _, err := o.fs.Stat(o.DockerfilePath); err != nil {
+			return err
+		}
+	}
+	if o.Schedule != "" {
+		_, err := cron.ParseStandard(o.Schedule)
+		if err == nil {
+			return nil
+		}
+		var rate time.Duration
+		if err != nil {
+			rate, err = time.ParseDuration(o.Schedule)
+		}
```

Comment:
```suggestion
		rate, err := time.ParseDuration(o.Schedule)
```

I think the previous `if err == nil` statement makes this if block unnecessary.